### ISSUE TITLE
Implement the kroki-default-options logic in the Ruby library

### ISSUE THREAD

**Issue by Mogztter** [Mogztter/asciidoctor-kroki#156]
Currently, this feature is only available in JavaScript: 

https://github.com/Mogztter/asciidoctor-kroki/blob/bca1ec83711989b0b1318222843eeac341e22c40/src/asciidoctor-kroki.js#L37-L59

**Comment by edl7878** [comment]
Can we use kroki in asciidoctor binary?

**Comment by Mogztter** [comment]
> Can we use kroki in asciidoctor binary?

Do you mean with the Asciidoctor (Ruby) CLI?
If so, I need to document it but you can use the `--require` (or `-r`) to require the Ruby library:

```
$ gem install asciidoctor-kroki
$ asciidoctor -r asciidoctor-kroki doc.adoc
```

The JavaScript and Ruby implementation have a few differences but we are trying to fill the gap.

> But when you use hugo with asciidoc, you need asciidoctor to render adoc files, at this time, if you want diagram, you can't use kroki, because there is no kroki plugin with asciidoctor binary.

Reading Hugo documentation, I think you should add `asciidoctor-kroki` in `extensions`: https://gohugo.io/content-management/formats/#external-helper-asciidoctor

If it's working we should probably update the documentation to state that `asciidoctor-kroki` is a possible value.

**Comment by edl7878** [comment]
Checked the Hugo doc, asciidoctor-kroki not in the possible extensions, for now, a proposal raised:

https://github.com/gohugoio/hugo/issues/7911

**Comment by edl7878** [comment]
I have made a PR of hugo which tries to make asciidoctor-kroki a possible extension.

https://github.com/gohugoio/hugo/pull/7919

**Comment by Mogztter** [comment]
Thanks @edl7878 :tada: 

I've created https://github.com/Mogztter/asciidoctor-kroki/issues/166 to document how to use the Ruby library. We could probably also explain how to integrate it in Hugo once your pull request is merged.
Feel free to add a comment in #166.

**Comment by edl7878** [comment]
Yes, if the pull request is merged, then the workflow is flawless, and I will explain how to integrate it in Hugo.
